kANe/v4/2_antyeShTi_Ashaucha_shuddhi/_index.md

```diff
@@ -3,6 +3,6 @@ title = "+2 Antyeṣṭi, Āśauca, Śuddhi"
 
 +++
 
-SECTION II 
+SECTION II
 
-ANTYESTI (rites after death ), SUDDHI ( purification from impurity due to death, birth and other causos) 
+ANTYEṢTI (rites after death ), ŚUDDHI ( purification from impurity due to death, birth and other causes)
```

Comment:
ANTYEṢTI should have टि

Reply:
means I should write ANTYEṢṬI like this?

Reply:
Yes

Reply:
corrected